jakarta.ls/src/main/java/io/microshed/jakartals/JakartaTextDocumentService.java

```diff
@@ -93,6 +93,21 @@ private void triggerValidationForAll() {
 		List<String> allDocs = documents.all().stream().map(doc -> doc.getUri()).collect(Collectors.toList());
 		triggerValidationFor(allDocs);
 	}
+	
+	
+	@Override
+	public CompletableFuture<Hover> hover(HoverParams params) {
+		// TODO give hover message to the server
+		LOGGER.info("Getting the hover going at server");
+		return jakartaLanguageServer.getLanguageClient().getJavaHover(params)
+				.thenApply(hover -> {
+					// if hover val is null
+					if (hover == null){
+						return null;
+					}
+					return hover;
+				});
+	}
```

Comment:
This code is unnecessary. It always just returns the value of `hover`.
```
.thenApply(hover -> {
					// if hover val is null
					if (hover == null){
						return null;
					}
					return hover;
				});
```

Can just be `return jakartaLanguageServer.getLanguageClient().getJavaHover(params)`